spring-boot/admin/src/main/java/eu/coatrack/admin/GitHubSecurityConfig.java

```diff
@@ -48,7 +41,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers(PERMITALL_RESOURCE_LIST).permitAll()
                 .anyRequest().authenticated()
                 .and()
-                .addFilterBefore(new PublicApiTokenAccessFilter(tokenServices), AbstractPreAuthenticatedProcessingFilter.class)
+             //   .addFilterBefore(new PublicApiTokenAccessFilter(tokenServices), AbstractPreAuthenticatedProcessingFilter.class)
```

Comment:
A reader of that comment would think: Why is that commented out? Is it important? Can I delete it? Idk ... etc. 

Imo, this appears to be "dead code" and should be deleted.

Reply:
this peace of code is from the ResourceServerTokenServices, so based on this, I will then create the follow-up branch and remove the code on this branch

Reply:
Okay. I suggest to create an issue about this and then this thread can be closed.

Reply:
Done